Bottom: 11987c07fbebad608967f966cf2f0c1136a64808
Top:    3f0afa425501b7381a471dd870205c9568ec5aea
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-21 20:55:17 +0000

Refresh of move-hid-drawing-api-prototype

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 6c6320a..c1de57b 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -1,5 +1,6 @@
 #include "global.h"
 #include "hid.h"
+#include "hid_draw.h"
 #include "polygon.h"
 
 static void
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index af719fd..2bbd2de 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -27,6 +27,7 @@
 
 #include "global.h"
 #include "hid.h"
+#include "hid_draw.h"
 #include "hid/common/hid_resource.h"
 
 #include "data.h"
